zeppelin-server/src/test/java/org/apache/zeppelin/ZeppelinIT.java

```diff
@@ -193,55 +194,68 @@ public void testAngularDisplay() throws InterruptedException{
       sleep(100, true);
 
       System.out.println("testCreateNotebook Test executed");
-    } catch (ElementNotVisibleException e) {
+    } catch (Exception e) {
       LOG.error("Exception in ZeppelinIT while testAngularDisplay ", e);
       File scrFile = ((TakesScreenshot)driver).getScreenshotAs(OutputType.FILE);
-
+      throw new Exception(e);
```

Comment:
can we instead say `throw e;` here? e is already an Exception
http://stackoverflow.com/questions/1097527/rethrowing-exceptions-in-java-without-losing-the-stack-trace